container/worker/run_openqa_worker.sh

```diff
@@ -6,4 +6,9 @@ if [[ -z $OPENQA_WORKER_INSTANCE ]]; then
 fi
 
 /root/qemu/kvm-mknod.sh
+
+kvm_original_group=$(ls -lh /dev/kvm | cut -d " " -f 4)
+groupmod -g $kvm_original_group kvm
```

Comment:
So my suspicion that the group IDs within and outside of the container don't match was correct. Note that `groupmod -g` expects a group ID but the ls command might return a group name in case there's actually a valid mapping. Hence you need the `-n` flag. I'd also change the quoting and get rid of the extra line:

```suggestion
groupmod -g "$(ls -lhn /dev/kvm | cut -d ' ' -f 4)" kvm
```

Reply:
Interesting.. because I tested this code and works. However I am going to include your suggestion to see if works too

Reply:
Of course "it works" … until it breaks when there's coincidentally a mapping for that group ID :-)

Reply:
Thanks, done and tested locally